index.js

```diff
@@ -8,7 +8,13 @@
   // Stub out `now` so we can use a more precise number in uuid generation, if
   // available.
   function now() {
-    return global.performance ? global.performance.now() : (new Date()).getTime();
+    if (global.performance && global.performance.now instanceof Function) {
```

Comment:
I prefer `typeof` checks for primitive types to `instanceof` (which doesn't work for things like strings, booleans, numbers), but it isn't in the styleguide explicitly. What was your reason to use `instanceof`?